index.js

```diff
@@ -39,6 +39,11 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
         const event = allEventsFromFetch[i];
         if (githubImportantEvents.indexOf(event.type) !== -1) {
             arrayOfImportantEvents.push(event);
+        } else if (event.payload) {
+            const typeWithAction = event.type + '.' + event.payload.action;
```

Comment:
I prefer using a template literal here, I think it's a bit more readable:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
`${event.type}.${event.payload.action}`